packages/experimental/src/components/AboutModal/AboutModal.js

```diff
@@ -0,0 +1,212 @@
+/**
+ * Copyright IBM Corp. 2020, 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  ComposedModal,
+  Link,
+  ModalHeader,
+  ModalFooter,
+  ModalBody,
+  Tabs,
+  Tab,
+} from 'carbon-components-react';
+
+import { expPrefix } from '../../global/js/settings';
+
+export const AboutModal = ({
+  copyrightText,
+  enableScroll,
```

Comment:
OK, I know Carbon do this in the modal but the idea of enable and disable for scroll seems bonkers to me. Can we lose this property and just scroll if needed.

Reply:
Just putting an update here, we talked on slack and decided to use `react-resize-detector`. I replaced the `enableScroll` prop with this package.